### PR TITLE
Adding integration tests for the new trusted calls (omni account)

### PR DESCRIPTION
**Pull request by silva-fj**
As topic, this PR adds integration tests for the omni account trusted calls as well as bug fixes and some clean up.

### CODE REVIEW

File: tee-worker/identity/client-api/parachain-api/prepare-build/interfaces/trusted_operations/definitions.ts [hunk 1]
```diff
@@ -25,7 +32,7 @@ export default {
         },
         TrustedCallAuthenticated: {
             call: "TrustedCall",
-            nonce: "Index",
+            nonce: "u32",
```
Comment (jonalvarezz) [inline tee-worker/identity/client-api/parachain-api/prepare-build/interfaces/trusted_operations/definitions.ts]: it should work with `Index` too. I mean, without changing anything else – did you find any issues? it'd be good to keep the type consistent with the other `nonce` in these definitions.

**Review by jonalvarezz — APPROVED** [review]

File: tee-worker/identity/client-api/parachain-api/prepare-build/interfaces/trusted_operations/definitions.ts (hunk 1 above)
Comment (silva-fj) [reply tee-worker/identity/client-api/parachain-api/prepare-build/interfaces/trusted_operations/definitions.ts]: Nope, there is no issue. That's why I changed it, to be consistent with the `TrustedCallSigned`:
```
TrustedCallSigned: {
    call: "TrustedCall",
    index: "u32",
    signature: "LitentryMultiSignature",
},
```
In the rust code the field `index` is called `nonce`

Comment (silva-fj) [reply tee-worker/identity/client-api/parachain-api/prepare-build/interfaces/trusted_operations/definitions.ts]: We can also change both to `Index`. What do you think?

Comment (Kailai-Wang) [reply tee-worker/identity/client-api/parachain-api/prepare-build/interfaces/trusted_operations/definitions.ts]: Historically it was called `index` in `frame_system::Config`, now it's `nonce`

**Review by Kailai-Wang — APPROVED** [review]
Great, thanks!
We are very close to switch-over, I guess only vc-issuance is left

File: tee-worker/identity/client-api/parachain-api/prepare-build/interfaces/trusted_operations/definitions.ts (hunk 1 above)
Comment (jonalvarezz) [reply tee-worker/identity/client-api/parachain-api/prepare-build/interfaces/trusted_operations/definitions.ts]: Right, I was confused by the enclave nonce's getter and `system.accountNextIndex` – both return `Index`.

> We can also change both to Index. What do you think?

Sounds good to me

**Comment by jonalvarezz** [comment]
Here is the client part: https://github.com/litentry/litentry-parachain/pull/3174

I used this branch as base. We may want to merge it first and then this one so it goes in the same commit.

I tested locally and it looks good

**Pull request merged**